elevator_entrance:
poses:
    first_floor:
        outside_elevator: # initial_pose
            x: 0
            y: 1.3
            Y: 1.719
        inside_elevator_first_try:
            x: -0.283
            y: 3.253
            Y: 1.57
#        inside_elevator_second_try: # same as - first_floor: inside_elevator
#            x: -0.15 
#            y: 3.3
#            Y: 0
    second_floor:
        outside_elevator: # initial_pose
            x: 7.0
            y: 4.1
            Y: 1.57
        inside_elevator_first_try:
            x: 6.55
            y: 6.35
            Y: 1.57
#        inside_elevator_second_try: # same as - second_floor: inside_elevator
#            x: 6.6
#            y: 6.4
#            Y: 0
exit_elevator:
    maps:
        first_floor: "/home/armadillo/catkin_ws/src/elevator_services/map/simulation_map_first_floor1.yaml"
        second_floor: "/home/armadillo/catkin_ws/src/elevator_services/map/simulation_map_elevator.yaml"
    poses:
        first_floor:
            inside_elevator: # initial_pose
                x: -0.15
                y: 3.3
                Y: 0
            outside_elevator:
                x: -0.433
                y: 0.714
                Y: -1.57
        second_floor:
            inside_elevator: # initial_pose
                x: 6.6
                y: 6.4
                Y: 0
            outside_elevator:
                x: 6.625
                y: 3.441
                Y: -1.57
